libs/product/driver/shopify/src/config/shopify-driver-config.ts

```diff
@@ -0,0 +1,9 @@
+export function shopifyDriverConfig(uri: string, token: string) {
```

Comment:
This function isn't related to `product` it looks like core shopify driver config.